ethernet_fabric.md

```diff
@@ -5,7 +5,7 @@ hosted in the lab.
 
 ## Switch & Fabric
```

Comment:
Can we also document the switch management host name and IPv4 address in this section as well. Don't publish the management password, but this just makes it easier to know how to reach the switch.